Formula/latexml.rb

```diff
@@ -0,0 +1,38 @@
+class Latexml < Formula
+  desc "LaTeX to XML/HTML/MathML Converter"
+  homepage "http://dlmf.nist.gov/LaTeXML"
+  url "https://github.com/brucemiller/LaTeXML/archive/v0.8.1.tar.gz"
+  sha256 "2ba1a580258ff5c7e3d9c2b40fd15cb4c92e388e5cd6b6127e8fbf9b1b9c63ce"
+
+  depends_on "cpanminus" => :recommended
+
+  def install
+    system "perl", "Makefile.PL", "PREFIX=#{prefix}"
+    system "make", "install"
+    doc.install "manual.pdf"
+  end
+
+  def caveats; <<-EOS.undent
+    When using images LaTeXML requires Image::Size, which
+    can be installed using cpanminus:
+      cpanm Image::Size
+
+    In case latexml can't locate LaTeXML/Core.pm:
+      export PERL5LIB=$PERL5LIB:#{HOMEBREW_PREFIX}/lib/perl5/site_perl
```

Comment:
This `PERL5LIB` stuff can be taken care of with a `write_env_script` call. See [how `creduce.rb` does it](https://github.com/Homebrew/homebrew-core/blob/a22826bf990df0557e138ecb51fc4eb2c07e3d7b/Formula/creduce.rb#L76).